app/views/lightweight_activities/show.html.haml

```diff
@@ -2,7 +2,7 @@
   = "#{@activity.name}"
 
 /
-  = @session_key
+  = @run_key
```

Comment:
Why is the run key added to the page here? I suspect this might have not be noticed because run_keys were not in the URLs of activity shows pages before.

Reply:
Oh well perhaps it would be useful for debugging it doesn't seem to hurt anything.